husky_extensions/itembox.js

```diff
@@ -490,6 +507,31 @@ define(function() {
             },
 
             /**
+             * Returns the icons of an item for given item data
+             *
+             * @param {Object} itemData The data of the item
+             * @returns {string} the html string of icons
+             */
+            getItemIcons: function(itemData) {
+                if (itemData[this.options.publishedStateName] === undefined) {
+                    return '';
+                }
+
+                var icons = '';
+                if (!itemData[this.options.publishedStateName] && !!itemData[this.options.publishedName]) {
+                    icons = templates.icons.notPublished(
+                        this.sandbox.translate(this.options.translations.unpublished)
+                    );
+                } else if (!itemData[this.options.publishedStateName]) {
+                    icons = templates.icons.notPublishedState(
+                        this.sandbox.translate(this.options.translations.unpublished)
+                    );
+                }
+
+                return icons;
+            },
```

Comment:
You surely mean the code in the smart-content. The thing is that the smart-content component does not use the itembox. This is an open issue.